### PR TITLE
Add hot key to reload content layer

### PR DESCRIPTION
**Pull request by ascorbic**
## Changes

Adds support for an "r + enter" hotkey to reload the content layer during dev. Also logs a hint at startup, showing how to display the hotkeys

image

### CODE REVIEW

File: packages/astro/src/core/messages.ts [hunk 1]
```diff
@@ -99,6 +99,8 @@ export function serverStart({
 		...localUrlMessages,
 		...networkUrlMessages,
 		'',
+		`${dim('Press')} ${bold('h + enter')} ${dim('to')} ${bold('show hot keys')}`,
+		''
```
Comment (bluwy) [inline packages/astro/src/core/messages.ts:103]: I don't think we want to show this message yet as proposed by Fred before, but an alternate tip display. ([Discord discussion](https://discord.com/channels/830184174198718474/1070208455329198190/1268119257263116401))

Comment (ascorbic) [reply packages/astro/src/core/messages.ts:103]: Doesn't that just apply to persistent footers though? This is just logged at the same time the server URLs are. 
image

Comment (bluwy) [reply packages/astro/src/core/messages.ts:103]: Yeah, it was also where I added the logs in the past, but we later discussed to not want to log it here for now.

Comment (ascorbic) [reply packages/astro/src/core/messages.ts:103]: Aah. I will remove it from this PR (which is probably sensible anyway as it'll be experimental) and will make the case with Fred that content layer tips the balance in favour of adding it once it's stable!

**Review by bluwy — APPROVED** [review]
I don't know if we would expand `r` in the future to also reload other things, or restart the server, but in any case I don't think it would be a breaking change.

**Comment by ematipico** [comment]
I would go for `g` instead of `r`. G would stand for "generate", in line with the sync command (which we will rename to gen)

**Comment by ascorbic** [comment]
But it's not generating anything - it's reloading the data

**Comment by ascorbic** [comment]
I've changed the key to "s" for "sync"

**Pull request merged**